### PR TITLE
Updated What's New with changes since November 4

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Updated the What's New with changes published since November 4th.

## Affected DevDocs pages

- https://devdocs.magento.com/whats-new.html

### CODE REVIEW

**Comment by hguthrie** [comment]
Can you look at the docker links in line 2670 and 2820?
I am not sure why they have `/reference/docker/`. Should it not be `/cloud/docker/` ?

**Comment by meker12** [comment]
> Can you look at the docker links in line 2670 and 2820?
> I am not sure why they have `/reference/docker/`. Should it not be `/cloud/docker/` ?

The links redirect to the correct path, but I replaced them so that the redirect is no longer required. Thanks for the update.

File: _data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,89 @@ description: |
...
+  date: November 19, 2019
+  link: https://github.com/magento/devdocs/pull/5929
+- description: Updated the [Release Notes](https://devdocs.magento.com/extensions/google-shopping-ads/release-notes/)
+    for the Google Shopping ads Channel v3.0.2 release.
+  versions: 3.0.2 Google Shopping ads
+  type: Major update
+  date: November 19, 2019
+  link: https://github.com/magento/devdocs/pull/6027
+- description: Added the "To change the directory from your module" section to the
+    [Configure Elasticsearch Stopwords](https://devdocs.magento.com/guides/v2.3/config-guide/elasticsearch/es-config-stopwords.html)
```
Comment (shrielenee) [inline _data/whats-new.yml]:
```suggestion
    [Configure Elasticsearch Stopwords](https://devdocs.magento.com/guides/v2.3/config-guide/elasticsearch/es-config-stopwords.html) topic
```

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**